.travis.yml

```diff
@@ -1,5 +1,5 @@
 language: java
-jdk: oraclejdk8
+jdk: openjdk9
```

Comment:
Why do we need to switch to Java9? I would prefer to stay on Java8 to make sure that this will always work with openHAB2, which is required to run on Java8.

Reply:
i would also prefer to keep java 8 but travis could not find oraclejdk8.. maybe i should try with openjdk8.  i will try it today evening.

Reply:
ok. openjdk8 works as well. so, pr is back on java8